src/components/piechart/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useMemo} from 'react';
+import _ from 'lodash';
+import View from '../view';
+import PartialCircle, {PartialCircleProps} from './PartialCircle';
+
+export type PieChartSegmentProps = Pick<PartialCircleProps, 'percentage' | 'color'>;
+
+export type PieChartProps = {
+  segments: PieChartSegmentProps[];
+  size?: number;
```

Comment:
consider adding a `defaultSegmentProps` where the user can control default UI for all segments like the divider color and width

Reply:
Cool, don't forget to add them also to the api.json